homeassistant/components/asuswrt/bridge.py

```diff
@@ -271,3 +317,163 @@ async def _get_temperatures(self) -> dict[str, Any]:
             raise UpdateFailed(exc) from exc
 
         return temperatures
+
+
+class AsusWrtHttpBridge(AsusWrtBridge):
+    """The Bridge that use HTTP library."""
+
+    def __init__(self, conf: dict[str, Any], session: ClientSession) -> None:
+        """Initialize Bridge that use HTTP library."""
+        super().__init__(conf[CONF_HOST])
+        self._api: AsusWrtHttp = self._get_api(conf, session)
+
+    @staticmethod
+    def _get_api(conf: dict[str, Any], session: ClientSession) -> AsusWrtHttp:
+        """Get the AsusWrtHttp API."""
+        return AsusWrtHttp(
+            conf[CONF_HOST],
+            conf[CONF_USERNAME],
+            conf.get(CONF_PASSWORD, ""),
+            use_https=conf[CONF_PROTOCOL] == PROTOCOL_HTTPS,
+            port=conf.get(CONF_PORT),
+            session=session,
+        )
+
+    @property
+    def is_connected(self) -> bool:
+        """Get connected status."""
+        return cast(bool, self._api.is_connected)
+
+    async def async_connect(self) -> None:
+        """Connect to the device."""
+        await self._api.async_connect()
+
+        # get main router properties
+        if mac := self._api.mac:
+            self._label_mac = format_mac(mac)
+        self._firmware = self._api.firmware
+        self._model = self._api.model
+
+    async def async_disconnect(self) -> None:
+        """Disconnect to the device."""
+        await self._api.async_disconnect()
+
+    async def async_get_connected_devices(self) -> dict[str, WrtDevice]:
+        """Get list of connected devices."""
+        try:
+            api_devices = await self._api.async_get_connected_devices()
+        except AsusWrtError as exc:
+            raise UpdateFailed(exc) from exc
+        return {
+            format_mac(mac): WrtDevice(dev.ip, dev.name, dev.node)
+            for mac, dev in api_devices.items()
+        }
+
+    async def async_get_available_sensors(self) -> dict[str, dict[str, Any]]:
+        """Return a dictionary of available sensors for this bridge."""
+        sensors_temperatures = await self._get_available_temperature_sensors()
+        sensors_cpu = await self._get_available_cpu_sensors()
+        sensors_types = {
+            SENSORS_TYPE_BYTES: {
+                KEY_SENSORS: SENSORS_BYTES,
+                KEY_METHOD: self._get_bytes,
+            },
+            SENSORS_TYPE_CPU: {
+                KEY_SENSORS: sensors_cpu,
+                KEY_METHOD: self._get_cpu_usage,
+            },
+            SENSORS_TYPE_MEMORY: {
+                KEY_SENSORS: SENSORS_MEMORY,
+                KEY_METHOD: self._get_memory_usage,
+            },
+            SENSORS_TYPE_RATES: {
+                KEY_SENSORS: SENSORS_RATES,
+                KEY_METHOD: self._get_rates,
+            },
+            SENSORS_TYPE_TEMPERATURES: {
+                KEY_SENSORS: sensors_temperatures,
+                KEY_METHOD: self._get_temperatures,
+            },
+        }
```

Comment:
Are Load Avg sensors not available for AsusWrtHttpBridge? Entity unique IDs are migrated but entities remain unavailable.
![obraz](https://github.com/home-assistant/core/assets/478555/f1dd4b0d-3616-4983-9bcb-c2962f3203d0)

Reply:
Yes, `AsusWrtHttpBridge` provide CPU usage instead of Load Avg. Because are not provided by integration, user should be able to manually delete using UI. I don't think we should do this automatically, what do you think?

Reply:
Then we definitely need to remove them from the registry if the integration uses the http/https protocol.

Reply:
Ok, I will manage this as migration action...

Reply:
Done.